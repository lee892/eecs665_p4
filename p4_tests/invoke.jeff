fn : (int arg1,bool arg2) int f{
}
fn : () void g {
	int a;
	f(a, true);
	g();
}
