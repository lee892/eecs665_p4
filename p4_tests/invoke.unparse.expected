fn : (int arg1,bool arg2) int f{
}

fn : () void g {
    int a;
    f{int,bool->int}(a{int}, true);
	g{->void}();
}
